### DISCUSSION TITLE
.env dosyasındaki NEXTAUTH_JWT_SECRET değişkenine ne yazmalıyım

### DISCUSSION THREAD

**Discussion by dilaraaydin** [harundogdu/twitter-clone#170] [Q&A]
JWT 'ın ne olduğunu anladım ama NEXTAUTH_JWT_SECRET değişkenine bu proje için ne yazmam gerektiğini bilmiyorum, yardımcı olur musunuz? şimdiden teşekkür ederim

**Comment by harundogdu** [comment]
@dilaraaydin Merhaba, bana discord'dan yazmanız mümkün mü acaba, oradan diğer env değişkenlerinizi de tanımlayayım. 

DC: harundogdu
